test/firestore_test.dart

```diff
@@ -47,6 +47,14 @@ Future main() async {
     expect(first.id, isNot(second.id));
   });
 
+  test('Get documents via collection query', () async {
+    var query = await firestore
```

Comment:
This test fails if there are no entries with that specific field and value.
It would fix the issue if you just create the value before testing it:
`await firestore.document('test/query').set({'test_field': 'test_value'});`

Reply:
Done